packages/gitbook/src/app/(site)/layout.tsx

```diff
@@ -0,0 +1,16 @@
+import { SpaceRootLayout } from '@/components/SpaceRootLayout';
```

Comment:
Should we name it `CustomizationRootLayout` instead?

It has nothing to do with a space really.